### PR TITLE
[TAS-2815] 🎨 Enable coupon code on all checkout pages

### PR DESCRIPTION
**Pull request by AuroraHuang22**

### CODE REVIEW

**Review by nwingt — COMMENTED** [review]
can refactor `getApplicableCoupon` to a mixin e.g. `couponMixin`?

**Comment by AuroraHuang22** [comment]
> [ ](/AuroraHuang22)
> ` [♻️ Refactor Christmas coupon logic into mixin](/likecoin/liker-land/pull/2000/commits/04b3e3901807b4389bad489cee03b0d3632c29be)`

[04b3e39](https://github.com/likecoin/liker-land/pull/2000/commits/04b3e3901807b4389bad489cee03b0d3632c29be)

File: src/pages/nft/class/_classId/index.vue [hunk 1]
```diff
@@ -426,24 +426,27 @@
 
 <script>
 import { mapActions, mapGetters } from 'vuex';
-import { nftClassCollectionType, parseNFTMetadataURL } from '~/util/nft';
-import { getNFTBookPurchaseLink, postNewStripeFiatPayment } from '~/util/api';
-import { logTrackerEvent, logPurchaseFlowEvent } from '~/util/EventLogger';
 import {
+  CHRISTMAS_CAMPAIGN_MIN_SPEND,
+  CHRISTMAS_CAMPAIGN_COUPON,
```
Comment (nwingt) [inline src/pages/nft/class/_classId/index.vue:431]: Unused?

Comment (AuroraHuang22) [reply src/pages/nft/class/_classId/index.vue]: [🔥 Remove unused code](https://github.com/likecoin/liker-land/pull/2000/commits/b15d252d6860452a1a1957ec6ab415a64f966517)

**Review by nwingt — APPROVED** [review]

**Pull request merged**